data/presets/fields/playground/max_age.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "max_age",
+    "type": "text",
```

Comment:
`min_age` and `max_age` can both be fields and have type `number`.  This will add up/down buttons to the side.